admin/src/js/controllers/main.js

```diff
@@ -12,7 +12,7 @@ angular.module('controllers').controller('MainCtrl',
     'ngInject';
     $translate.use('en');
     $scope.authorized = false;
-    Auth('can_configure')
+    Auth.any([['can_configure'], ['can_view_outgoing_messages'], ['can_export_all']])
```

Comment:
I don't think this is meant to be an array of arrays, is it? I think it should be...

```js
Auth.any(['can_configure', 'can_view_outgoing_messages', 'can_export_all'])
```

Reply:
I thought so as well, only to realize it was wired to working in a unique way, in order to avoid tampering with the functionality and introducing bugs I just followed how `mm-auth-any` directive does it and ended up with the array of arrays

Reply:
Ahh yeah I see now. Could you please add JSDoc to the Auth.any function so it's obvious for the next reader?